docs/user-guide/debugging-pcap.md

```diff
@@ -7,7 +7,7 @@ A packet capture allows for inspecting the contents of every packet transmitted
 Since QUIC is an encrypted transport protocol, the payload of each packet is not readable in a standard packet capture. `s2n-quic` supports exporting the TLS session keys used by each QUIC connection so that the packet capture may be decrypted. Both the `s2n-tls` and `rustls` TLS providers support key logging through their associated builders:
 
 ```rust
-let tls = s2n_quic::provider::tls::default::Serverhell::builder()
+let tls = s2n_quic::provider::tls::default::Serverhello::builder()
```

Comment:
it actually should just be `Server`

```suggestion
let tls = s2n_quic::provider::tls::default::Server::builder()
```

Reply:
https://docs.rs/s2n-quic/latest/s2n_quic/provider/tls/default/struct.Server.html#method.builder